server/index.js

```diff
@@ -22,6 +22,9 @@ const db = require("./lib/in-memory-db");
 // require it and pass the `db` parameter immediately:
 const DataHelpers = require("./lib/data-helpers.js")(db);
 
+// Update the tweets in data-files/initial-tweets.json
+const dateAdjust = require("./lib/date-adjust")();
```

Comment:
Are you using `dateAdjust` anywhere? Do you need to set this to a variable?

Reply:
Good catch, thank you and removed.

Reply:
Just being pedantic, but it's not actually a self invoking function. That would be an [IIFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE). Here, we're simply calling the function in the same line that it's required.

You can just leave the comment about what the required function is doing and remove the `const dateAdjust =` part.

Reply:
Good to know - thank you, and done!